tools/hangar/__snapshots__/test_corpus/add_consumer.w_compile_tf-aws.md

```diff
@@ -393,6 +393,7 @@ class $Root extends $stdlib.std.Resource {
       constructor(scope, id, ) {
         super(scope, id);
         this._addInflightOps("handle");
+        this.display.hidden = true;
```

Comment:
We can probably "shift right" some of this logic into the SDK if this gets too bloated

Reply:
We inherit from a `HiddenResource` for example.. But I guess this is ok.